lib/web/integrations_awsoidc.go

```diff
@@ -930,7 +930,7 @@ func (h *Handler) awsOIDCCreateAWSAppAccess(w http.ResponseWriter, r *http.Reque
 
 	getUserGroupLookup := h.getUserGroupLookup(r.Context(), clt)
 
-	publicAddr := libutils.DefaultAppPublicAddr(integrationName, h.PublicProxyAddr())
+	publicAddr := app.DefaultAppPublicAddr(integrationName, h.proxyDNSName())
```

Comment:
This prevents AWS console app resources from having a port in the public address. Prior to this change this was the only place an app public address was created with `h.PublicProxyAddr()` instead of `h.proxyDNSName()`.

Reply:
After this change, what happens if the proxy is not running on port `:443`? Does it still work?